### PR TITLE
fix outdated examples with new field names

### PR DESCRIPTION
**Pull request by matthewhanson**
**Related Issue(s):** #

None, fixes some examples with new field names from the sat extension that were missed

**Proposed Changes:**

1. fix examples
2. 

**PR Checklist:**

- [x] This PR has **no** breaking changes.
- [x] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-spec/blob/dev/CHANGELOG.md) **or** a CHANGELOG entry is not required.
- [x] API only: I have run `npm run generate-all` to update the [generated OpenAPI files](https://github.com/radiantearth/stac-spec/blob/dev/api-spec/README.md#openapi-definitions).

### CODE REVIEW

**Comment by matthewhanson** [comment]
@m-mohr  Ok, fixed those - and also fixed some old examples that still used the `dtr` prefix for datetime-range

File: item-spec/examples/digitalglobe-sample.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "sat:sun_azimuth_angle": 168.7,
     "eo:cloud_cover": 50,
     "sat:off_nadir_angle": 18.4,
-    "eo:azimuth": 34.5,
+    "sat:azimuth": 34.5,
```
Comment (m-mohr) [inline item-spec/examples/digitalglobe-sample.json]: azimuth_angle

File: extensions/single-file-stac/examples/example-search.json [hunk 2]
```diff
@@ -40,8 +40,8 @@
             },
             "properties": {
                 "datetime": "2018-02-08T18:02:15.719478+00:00",
-                "eo:sun_azimuth": 152.63804142,
-                "eo:sun_elevation": 31.82216637,
+                "sat:sun_azimuth_angle": 152.63804142,
+                "sat:sun_elevation": 31.82216637,
```
Comment (m-mohr) [inline extensions/single-file-stac/examples/example-search.json]: sun_elevation_angle

File: item-spec/examples/digitalglobe-sample.json (hunk 1 above)
Comment (matthewhanson) [reply item-spec/examples/digitalglobe-sample.json]: Hmm, that's fixed on my side and all pushed, try refreshing?

Comment (matthewhanson) [reply item-spec/examples/digitalglobe-sample.json]: oh wait, no it's not...sorry, more fixes coming

**Pull request merged**